WordPress/Classes/Services/AccountService.h

```diff
@@ -3,7 +3,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class WPAccount, Blog;
+@class WPAccount, AbstractPost;
```

Comment:
Nice one on removing `Blog`. Curious about the plan for `AbstractPost`?

Reply:
Whoops. I should've gotten rid of that. Leftover from when I was playing with having `needsVerificationBeforePublishing:` (or similar) in here.

Reply:
Oh no worries!  I thought it might be a spoiler of things to come :D